### PR TITLE
Handle failure for the right variable

### PR DESCRIPTION
**Pull request by katiecrane**
#### What's this PR do?
Use `$update` here because that's the area we are in.

#### How should this be reviewed?
Brief 👀 

#### Any background context you want to provide?
I made a typo.

#### Checklist
- [ ] Tested on staging.

### CODE REVIEW

File: scripts/rogue-keeper-upper.php [hunk 1]
```diff
@@ -187,12 +190,18 @@
 
       // Put request in failed table for future investigation
       // @TODO: only put in this table if it's not already there
-      dosomething_rogue_handle_migration_failure($data, $post->sid, $post->rbid, $post->fid, $response, $e);
+      dosomething_rogue_handle_migration_failure($data, $update->sid, $update->rbid, $update->fid, $response, $e);
+
+        // Update the timestamp so we only check for updates after where we left off
```
Comment (DFurnes) [inline scripts/rogue-keeper-upper.php]: Indentation! 😇

File: scripts/rogue-keeper-upper.php [hunk 2]
```diff
@@ -187,12 +190,18 @@
...
+        variable_set('dosomething_rogue_last_timestamp_sent', $update->timestamp);
     }
     catch (DoSomething\Gateway\Exceptions\ApiException $e) {
       echo 'api exception' . PHP_EOL;
       // Put request in failed table for future investigation
-      dosomething_rogue_handle_migration_failure($data, $post->sid, $post->rbid, $post->fid, $response, $e);
+      dosomething_rogue_handle_migration_failure($data, $update->sid, $update->rbid, $update->fid, $response, $e);
+
+        // Update the timestamp so we only check for updates after where we left off
```
Comment (DFurnes) [inline scripts/rogue-keeper-upper.php]: Also here! 👾

**Review by DFurnes — APPROVED** [review]

**Comment by katiecrane** [comment]
@DFurnes updated to set the system variables in fewer places!

**Review by DFurnes — APPROVED** [review]
Aw yiss, ship it! ⚓️

**Pull request merged**